packages/common/src/directives/styling_differ.ts

```diff
@@ -212,11 +212,10 @@ function bulidMapFromValues(
     // case 1: map
     for (let i = 0; i < keys.length; i++) {
       let key = keys[i];
-      key = trim ? key.trim() : key;
       const value = (values as{[key: string]: any})[key];
```

Comment:
Nit: Can we add a comment here that the map has untrimmed values? e.g.

```suggestion
      // Map uses untrimmed keys, so don't trim until passing to `setMapValues`
      const value = (values as{[key: string]: any})[key];
```